apps/leo_gateway/include/leo_gateway.hrl

```diff
@@ -445,6 +445,25 @@
               })
             %% ?notify_metrics(<<"PUT">>,_Bucket,_Size)
         end).
+-define(access_log_copy(_Bucket,_Path,_Size,_Response,_Begin),
+        begin
+            _Latency = erlang:round((leo_date:clock() - _Begin) / 1000),
```

Comment:
Assign a variable to leo_date:clock and reference it here and the following code where leo_date:clock is called in order to reduce os:timestamp calls.